lib/matplotlib/cbook/__init__.py

```diff
@@ -2128,3 +2134,15 @@ def _check_not_matrix(**kwargs):
     for k, v in kwargs.items():
         if isinstance(v, np.matrix):
             raise TypeError(f"Argument {k!r} cannot be a np.matrix")
+
+
+def _check_in_list(values, **kwargs):
+    """
+    For each *key, value* pair in *kwargs*, check that *value* is in *values*;
+    if not, raise an appropriate ValueError.
+    """
```

Comment:
Even though this is just internal, I would like an explanatory example in the docstring because the kwarg usage is a bit special. Maybe even with a note that you'll  usually do `_check_in_list(values, param=param)`.